src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write code here
+document.addEventListener('DOMContentLoaded', () => {
+  const tree = document.querySelectorAll('.tree li');
+
+  tree.forEach((item) => {
+    const span = document.createElement('span');
+
+    span.textContent = item.childNodes[0].nodeValue.trim();
+
+    item.insertBefore(span, item.childNodes[0]);
+    item.childNodes[1].nodeValue = '';
+
+    const childUl = item.querySelector('ul');
```

Comment:
The `childUl` variable is defined outside the click event listener, which is fine. However, it might be more efficient to check if `childUl` exists inside the event listener to avoid unnecessary DOM queries when the span is clicked but there is no child `ul`.